.github/workflows/playwright.yml

```
@@ -0,0 +1,52 @@
+          - 6379:6379
+    steps:
+    - uses: actions/checkout@v4
+    - uses: actions/setup-node@v4
+      with:
+        node-version: lts/*
+    - name: Install dependencies
+      run: npm ci
+    - name: Install Playwright Browsers
+      run: npx playwright install --with-deps
+    - name: Set up SSH
+      run: |
```

actionlint

shellcheck reported issue in this script: SC2046:warning:1:6: Quote this to prevent word splitting

(shellcheck)